src/chains/ethereum/ethereum/src/blockchain.ts

```diff
@@ -1284,7 +1298,14 @@ export default class Blockchain extends Emittery<BlockchainTypedEvents> {
       this.emit("ganache:vm:tx:before", {
         context: transactionEventContext
       });
-      await vm.runTx({ tx, block: block as any });
+      await vm.runTx({
+        skipHardForkValidation: true,
+        skipNonce: true,
+        skipBalance: true,
+        skipBlockGasLimitValidation: true,
```

Comment:
We don't need to check these since we've already run all these transactions before, these validations shouldn't be needed.